.github/workflows/makefile.yml

```
@@ -40,6 +40,12 @@ jobs:
         path: .lake
         key: ${{ runner.os }}-lake
 
+    - name: Build AWS-LC and running ELFSage tests (Ubuntu)
+      if : ${{ runner.os != 'macOS' }}
+      run: |
+        ./scripts/top.sh
```

```suggestion
        ./scripts/top.sh
```

Sorry, one last request, as I just noticed: Can I have an explanation for why the script is called `top.sh`? Perhaps what I prefer would be to rename the script to `ci_macos_aws_lc_setup.sh` or something equally verbose :)

---

Good suggestion. I will update it to `ci_ubuntu_build_awslc.sh`.